### DISCUSSION TITLE
[Esame 29 Gennaio 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#265] [Esami - Bertini @ Informatica]
**Esercizio 1.** Da un mazzo di carte napoletane (40 carte divise in 4 semi) si estraggono consecutivamente (senza rimpiazzo) quattro carte.

**(1)** Calcolare la probabilità che la seconda carta estratta sia di denari.

**(2)** Sapendo che la seconda carta estratta è di denari, calcolare la probabilità che anche la prima estratta sia di denari.

**(3)** Calcolare la probabilità di estrarre, nell'ordine, un asso, un due, un tre e un quattro.

**(4)** Calcolare la probabilità di estrarre un asso, un due, un tre e un quattro (indipendentemente dall'ordine in cui sono stati estratti).

**Comment by cornflexxx** [comment]
1)

$$ \mathbb{P}(\text{la 2° è denari}) = \mathbb{P}(\text{la 2° è denari e la 1° non è denari})+ \mathbb{P}(\text{la 2° è denari e la 1°  è denari})=3/4\cdot \frac{10}{39}+1/4\cdot \frac{9}{39}= 1/4$$

2)

$$\mathbb{P}(\text{1° D | 2° D})= \frac{\mathbb{P}(1° D \cap 2°D)}{\mathbb{P}(2°D)}= 9/39$$

3)

$$\mathbb{P}( 1 2  3  4) = \frac{4^4}{40\cdot 39 \cdot 38 \cdot 37}$$

4) Moltiplico il risultato precedente per il numero di permutazioni : 

$$\mathbb{P}( 1 2  3  4) \cdot 4! $$

**Comment by Traster-uni** [comment]
![soluz_es1_esameGennaio_ProfBertini](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/84441861/7776e8ab-9f3e-453c-8655-ec1e1a38f1a6)
